test/e2e/snaps/test-snap-cronjob.spec.js

```diff
@@ -33,34 +28,30 @@ describe('Test Snap Cronjob', function () {
         // added delay for firefox (deflake)
         await driver.delayFirefox(1000);
 
-        // wait for and click connect
-        await driver.waitForSelector('#connectcronjobs');
+        // click connect
         await driver.clickElement('#connectcronjobs');
```

Comment:
:information_source:  whenever we use `clickElement method, we don't need to use `waitForSelector` on the same selector, because this is already done inside the clickElement (it waits until the element is there and `enabled`  before clicking)